CompareGraphs.py

```diff
@@ -95,9 +167,15 @@
 
     for iObj, (inObj, legend) in enumerate(zip(inObjs, legends)):
         if isinstance(inObj, TGraph):
-            inObj.Draw('same p')
+            if('drawoptsyst' in plot['input'][iObj]):
+                inObj.Draw('same' + plot['input'][iObj]['drawoptsyst'])
+            else:
+                inObj.Draw('same ' + drawOpts[iObj])
+
+        if isinstance(inObj, TF1):
+            inObj.DrawF1(fx1, fx2, "same")
```

Comment:
fx1 and fx2 are not defined. Please check

Reply:
They are defined as the xmin and xmax values of the pad when the `DrawFrame` method is called